src/app/(proper_react)/redesign/(public)/PlansTable.tsx

```diff
@@ -227,6 +227,7 @@ export const PlansTable = (props: Props) => {
                       elems: { b: <b /> },
                     },
                   )}
+                  &nbsp;
```

Comment:
I added these because the trigger buttons were flush with the text when they were on the same line.